.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,49 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug Report
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: textarea
+
+    attributes:
+      description: Concise, complete, and comprehesnsive description.
```

A description like below could remove the need for a placeholder here.

```suggestion
      description: Describe the issue in detail. Consider including code examples or screenshots where appropriate.
```